examples/eulerflow_profiling.py

```diff
@@ -0,0 +1,229 @@
+__copyright__ = """
+Copyright (C) 2020 University of Illinois Board of Trustees
+"""
+
+__license__ = """
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
My understanding is that array context support for Euler will happen in #7, and `mtc/array-context` has sort of fallen by the wayside.

@MTCam, correct?

Reply:
This was just due to merging with the other array context branch I believe. Removed.